storage/engine/rocksdb/db.cc

```diff
@@ -751,27 +742,7 @@ class DBMergeOperator : public rocksdb::MergeOperator {
     }
 
     for (int i = 0; i < operand_list.size(); i++) {
-      if (!MergeOne(&meta, operand_list[i], true, logger)) {
-        return false;
-      }
-    }
-
-    if (!meta.SerializeToString(new_value)) {
-      rocksdb::Warn(logger, "serialization error");
-      return false;
-    }
-    return true;
-  }
-
-  virtual bool PartialMergeMulti(
```

Comment:
This code is called from RocksDB. It definitely isn't dead code that can just be removed.